Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PlannerTaskList/PlannerTaskListPage.xaml

```diff
@@ -0,0 +1,22 @@
+﻿<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.PlannerTaskListPage"
```

Comment:
Got this error in sample app?

![image](https://user-images.githubusercontent.com/24302614/42008459-8138b2ea-7a3a-11e8-89c5-b44a72d74302.png)

Reply:
We're waiting for @nmetulev to fix our test account to have the proper permissions.  So I just tried my regular old MSA account.  After logging in with that was where I saw the two problems here.

Reply:
@michael-hawker Unfortunately, all of our graph controls accept work or school account only, MSA isn't supported.